### PR TITLE
talisman check for pre-commit and pre-push hooks

### PR DESCRIPTION
**Pull request by PoojaBabusing**
This PR is for issue: https://mosip.atlassian.net/browse/INJI-413

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
+    "jetify": "jetify",
     "start": "react-native start",
     "android:mosip": "react-native run-android --variant=mosipDebug",
     "ios": "react-native run-ios",
     "build:android:mosip": "cd android && ./gradlew :app:assembleMosipRelease && cd ..",
     "i18n:compile-strings": "node scripts/compile-strings.js",
     "lint": "eslint . --ext .js,.jsx,.ts,.tsx --fix",
     "test": "jest",
-    "postinstall": "patch-package"
+    "postinstall": "patch-package && yarn jetify && sh tools/talisman/talisman-precommit.sh"
```
Comment (tilak-puli) [inline package.json:12]: can we use npm instead of yarn as we use npm by default

File: tools/talisman/talisman-precommit.sh [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    fi
+
+    if [ -f $prePushFile ] && [ -z $GITHUB_ACTIONS ]
+    then
+        rm $prePushFile
+    fi
+
+    if [ ! -f $preCommitFile ] && [ -z $GITHUB_ACTIONS ]
+    then
+        curl https://thoughtworks.github.io/talisman/install.sh > ~/install-talisman.sh
```
Comment (tilak-puli) [inline tools/talisman/talisman-precommit.sh:22]: @swatigoel we are downloading talisman install file from thoughtworks github.

this should fine right? if we don't want to do this then we have store shell script in our repo

File: package.json (hunk 1 above)
Comment (PoojaBabusing) [reply package.json]: Done replaced with npm run jetify

File: tools/talisman/talisman-precommit.sh (hunk 2 above)
Comment (swatigoel) [reply tools/talisman/talisman-precommit.sh:22]: Let it be downloading from thoughtworks github. It's open source, so should not be worry

**Review by tilak-puli — APPROVED** [review]

**Review by swatigoel — APPROVED** [review]

**Pull request merged**